cli/js/lib.deno.ns.d.ts

```diff
@@ -317,8 +317,8 @@ declare namespace Deno {
    * @param options Can be used to tune size of the buffer. Default size is 32kB
    */
   export function copy(
-    src: Reader,
-    dst: Writer,
+    src: Reader | ReadableStream,
+    dst: Writer | WritableStream,
```

Comment:
Again, what is the intuition behind just `copy()` supporting these and not every other IO function? This shouldn't be included in favour of just the conversion utilities.

Reply:
Wanted to do progressive changes instead of a massive PR (Initially it was meant to be an issue to discuss the API, but since I had some code that I was testing submitted a PR instead). 

I'm good with only having utilities to convert from/to and leave `Deno.copy` as is.

Now If we do only utilities, should they be in the core or in std?

Reply:
We have `Deno.iter()` so I think these are candidates to be included in the CLI.

Reply:
It's certainly handy for copy to take ReadableStream/WritableStream but in the interest of being conservative in our API changes, I think toReader and toWriter would solve the problem for now. Let's do the most minimal thing and expand on it later.

That is, I agree with @nayeemrmn, please undo this change.

Reply:
I'm working on it and now I'm wondering whether we want to make the API take a `ReadableStream` or a `ReadableStreamDefaultReader / ReadableStreamBYOBReader`, the same goes for `WritableStream`

Reply:
The latter is the logical and minimal choice. It's one more method call to use on a `*ableStream`.

Reply:
I believe the reader/writer interfaces are what we want to use (the latter).